clients/src/main/java/org/apache/kafka/clients/consumer/MockConsumer.java

```diff
@@ -446,8 +447,8 @@ public synchronized void resume(Collection<TopicPartition> partitions) {
     }
 
     @Override
-    public final synchronized void close() {
-        this.closed = true;
+    public synchronized void close() {
```

Comment:
Should we move the `synchronized` to the other `close` method or add it there as well?

Reply:
Makes sense, done.